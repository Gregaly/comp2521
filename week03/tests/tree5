 24
  9
 45
 29
 59
 71
 38
 34
 80
 60
 76
 99
 33
 90
 15
 91
 22
 94
 26
 25
 19
 87
 86
  4
 42
 74
 37
 50
 57
 32
 55
 92
 89
 93
 30
 58
 47
100
 85
 96
 35
 63
 10
  6
 21
  5
 73
 82
 62
 31
 81
 84
  7
 68
 67
 49
 53
 18
 48
  8
 40
 88
 17
 70
 97
 13
 23
 41
 83
 36
 46
 54
 66
 72
 52
 28
 12
 27
 65
  3
 64
 98
 75
 16
 79
 11
 43
 95
 77
 69
 61
 56
 20
 39
 14
 78
  1
 44
 51
  2
