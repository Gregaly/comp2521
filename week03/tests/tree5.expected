Tree:
                                24
                                / \
                               /   \
                              /     \
                             /       \
                            /         \
                           /           \
                          /             \
                         /               \
                        /                 \
                       /                   \
                      /                     \
                     /                       \
                    /                         \
                   /                           \
                  /                             \
                 /                               \
                /                                 \
               /                                   \
              /                                     \
             /                                       \
            9                                        45
           / \                                       / \
          /   \                                     /   \
         /     \                                   /     \
        /       \                                 /       \
       /         \                               /         \
      /           \                             /           \
     /             \                           /             \
    4              15                         /               \
   / \             / \                       /                 \
  3   6           /   \                     /                   \
 /   / \         /     \                   /                     \
1   5   7       /       \                 /                       \
 \       \     /         \               /                         \
  2       8   /           \             /                           \
             10           22           /                             \
              \           / \         /                               \
              13         /   \       /                                 \
              / \       19   23     /                                   \
             /   \     / \         29                                   59
            12   14   /   \       / \                                   / \
           /         18   21     /   \                                 /   \
          11        /     /     /     \                               /     \
                   17    20    /       \                             /       \
                  /           /         \                           /         \
                 16          26         38                         /           \
                            / \         / \                       /             \
                           /   \       /   \                     /               \
                          25   28     /     \                   /                 \
                               /     /       \                 /                   \
                              27    /         \               50                   71
                                   34         42             / \                   / \
                                  / \         / \           /   \                 /   \
                                 /   \       /   \         /     \               /     \
                                33   37     40   43       /       \             /       \
                               /     /     / \     \     /         \           /         \
                              32    35    /   \    44   47         57         /           \
                             /       \   39   41       / \         / \       /             \
                            30       36               /   \       /   \     /               \
                             \                       46   49     55   58   /                 \
                             31                           /     / \       /                   \
                                                         48    /   \     60                   80
                                                              53   56     \                   / \
                                                             / \          63                 /   \
                                                            /   \         / \               /     \
                                                           52   54       /   \             /       \
                                                          /             62   68           /         \
                                                         51            /     / \         /           \
                                                                      61    /   \       /             \
                                                                           67   70     76             99
                                                                          /     /     / \             / \
                                                                         66    69    /   \           /   \
                                                                        /           /     \         90   100
                                                                       65          74     79       / \
                                                                      /           / \     /       /   \
                                                                     64          /   \   77      /     \
                                                                                73   75   \     87     91
                                                                               /          78   / \       \
                                                                              72              /   \      94
                                                                                             86   89     / \
                                                                                            /     /     /   \
                                                                                           85    88    /     \
                                                                                          /           92     96
                                                                                         82            \     / \
                                                                                        / \            93   /   \
                                                                                       /   \               95   97
                                                                                      81   84                     \
                                                                                           /                      98
                                                                                          83

#nodes = 100,  #leaves = 34
Infix  : 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63 64 65 66 67 68 69 70 71 72 73 74 75 76 77 78 79 80 81 82 83 84 85 86 87 88 89 90 91 92 93 94 95 96 97 98 99 100 
Prefix : 24 9 4 3 1 2 6 5 7 8 15 10 13 12 11 14 22 19 18 17 16 21 20 23 45 29 26 25 28 27 38 34 33 32 30 31 37 35 36 42 40 39 41 43 44 59 50 47 46 49 48 57 55 53 52 51 54 56 58 71 60 63 62 61 68 67 66 65 64 70 69 80 76 74 73 72 75 79 77 78 99 90 87 86 85 82 81 84 83 89 88 91 94 92 93 96 95 97 98 100 
Postfix: 2 1 3 5 8 7 6 4 11 12 14 13 10 16 17 18 20 21 19 23 22 15 9 25 27 28 26 31 30 32 33 36 35 37 34 39 41 40 44 43 42 38 29 46 48 49 47 51 52 54 53 56 55 58 57 50 61 62 64 65 66 67 69 70 68 63 60 72 73 75 74 78 77 79 76 81 83 84 82 85 86 88 89 87 93 92 95 98 97 96 94 91 90 100 99 80 71 59 45 24 
ByLevel: 24 9 45 4 15 29 59 3 6 10 22 26 38 50 71 1 5 7 13 19 23 25 28 34 42 47 57 60 80 2 8 12 14 18 21 27 33 37 40 43 46 49 55 58 63 76 99 11 17 20 32 35 39 41 44 48 53 56 62 68 74 79 90 100 16 30 36 52 54 61 67 70 73 75 77 87 91 31 51 66 69 72 78 86 89 94 65 85 88 92 96 64 82 93 95 97 81 84 98 83 
